src/test/java/kitchenpos/common/TestFixture.java

```diff
@@ -0,0 +1,45 @@
+package kitchenpos.common;
+
+import kitchenpos.domain.MenuProduct;
+import kitchenpos.domain.Order;
+import kitchenpos.domain.OrderLineItem;
+import kitchenpos.domain.OrderStatus;
+import kitchenpos.domain.OrderTable;
+import kitchenpos.domain.Product;
+
+public class TestFixture {
+	public static final String 예제_메뉴그룹명 = "인도요리";
+	public static final String 예제_상품명 = "피자";
+	public static final int 예제_상품_가격 = 15000;
+	public static final long 주문대상_테이블ID = 7L;
+	public static final long 빈_테이블ID = 1L;
+	public static final long 존재하지_않는_테이블ID = 999L;
+	public static final long 예제_테이블_그룹_ID = 1L;
+	public static final long 조리상태인_테이블그룹_ID = 2L;
+	public static final int 후라이드_가격 = 16000;
```

Comment:
```suggestion
	public static final int 후라이드_가격 = 16_000;
```
별건 아니지만 숫자리터럴을 사용할 때 언더스코어를 사용하면 가독성을 향상 시킬수 있어요.

Reply:
피드백 감사합니다. 수정반영하겠습니다.